### PR TITLE
fix illegal instruction on platform which has avx only

### PR DESCRIPTION
**Pull request by LHT129**
fixed: #176

### CODE REVIEW

**Review by ShawnShawnYou — APPROVED** [review]

File: src/simd/CMakeLists.txt [hunk 1]
```diff
@@ -50,7 +55,7 @@ endmacro ()
 
 simd_add_definitions (DIST_CONTAINS_SSE -DENABLE_SSE=1)
 simd_add_definitions (DIST_CONTAINS_AVX -DENABLE_AVX=1)
-simd_add_definitions (DIST_CONTAINS_AVX2 -DENABLE_AVX2=1)
+#simd_add_definitions (DIST_CONTAINS_AVX2 -DENABLE_AVX2=1)
```
Comment (jiaweizone) [inline src/simd/CMakeLists.txt:58]: remove unused line

File: src/simd/normalize.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// limitations under the License.
+
+#include "normalize.h"
+
+#include "simd_status.h"
+
+namespace vsag {
+
+static NormalizeType
+SetNormalize() {
```
Comment (jiaweizone) [inline src/simd/normalize.cpp:23]: GetNormalize ?

File: src/simd/normalize.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#if defined(ENABLE_SSE)
+        return sse::Normalize;
+#endif
+    }
+    return generic::Normalize;
+}
+NormalizeType Normalize = SetNormalize();
+
+static DivScalarType
+SetDivScalar() {
```
Comment (jiaweizone) [inline src/simd/normalize.cpp:42]: GetDivScalar ?

Comment (LHT129) [reply src/simd/normalize.cpp:42]: Get is better, but all simd define use Set now, I'd like to fix it in another PR

File: src/simd/normalize.cpp (hunk 2 above)
Comment (LHT129) [reply src/simd/normalize.cpp:23]: ditto

File: src/simd/CMakeLists.txt (hunk 1 above)
Comment (LHT129) [reply src/simd/CMakeLists.txt:58]: we have a very useful annotation for this line to fix the avx bug, so shound not remove this now 
image

File: src/simd/normalize.cpp (hunk 3 above)
Comment (jiaweizone) [reply src/simd/normalize.cpp:42]: OK

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**